src/science/mathematics/big-integers.md

```diff
@@ -0,0 +1,30 @@
+# Miscellaneous
+
+## Big integers
+
+[![num-badge]][num] [![cat-science-badge]][cat-science]
+
+If we need to make calculations on integers exceeding size of standard integer type, we can use BigInt.
```

Comment:
I think we should link to [BigInt](https://docs.rs/num-bigint/0.2.0/num_bigint/struct.BigInt.html) here.